bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -2116,7 +2153,7 @@ public long internal_new_GC (GCData data) {
 		}
 		data.device = device;
 		data.image = this;
-		data.font = device.systemFont;
+		data.font = Font.win32_new(device.getSystemFont(), this.currentDeviceZoom);
```

Comment:
Just for my understanding: at most places fonts are rescaled to the native device zoom. Why is the current device zoom used in this case?

Reply:
Resolved as changed to native zoom